client/src/components/ProfileDetail/RequestCard.tsx

```diff
@@ -0,0 +1,126 @@
+import DateFnsUtils from '@date-io/date-fns';
+import { Box, Button, CircularProgress, Grid, TextField, Typography } from '@material-ui/core';
+import { Rating } from '@material-ui/lab';
+import { MuiPickersUtilsProvider, KeyboardDatePicker, KeyboardTimePicker } from '@material-ui/pickers';
+import { useState } from 'react';
+import useStyles from './useStyles';
+
+interface Props {
+  profile: {
+    priceRate: number;
+  };
+}
+export default function RequestCard({ profile }: Props): JSX.Element {
+  const classes = useStyles();
+  const currentDate = new Date();
+  const [startDate, setStartDate] = useState<Date | null>(currentDate);
+  const [endDate, setEndDate] = useState<Date | null>(currentDate);
+  const [serviceHours, setServiceHours] = useState<number>(0);
+  const [sendSuccess, setSendSuccess] = useState<boolean>(false);
+
+  const handleStartDate = (date: Date | null) => {
+    if (date?.getTime() !== undefined && startDate?.getTime() !== undefined && date?.getTime() < startDate?.getTime()) {
+      alert('Please select today or future day or future time');
```

Comment:
It is good that you handle this case also,  there is an easy way to handle this. Just pass `minDate` prop to `KeyboardDatePicker` to disable previous dates.
```
import { format } from 'date-fns';
minDate={format(new Date(), 'yyyy-MM-dd')}

```